iwyu.cc

```diff
@@ -3997,6 +3999,22 @@ int main(int argc, char **argv) {
   std::unique_ptr<clang::CompilerInstance> compiler(CreateCompilerInstance(
       options_parser.clang_argc(), options_parser.clang_argv()));
   if (compiler) {
+    // Figure out the absolute path to the resource-dir, which might be relative
+    // to the install directory
+    StringRef Executable = argv[0];
+    llvm::SmallString<128> AbsoluteResourceDir(IWYU_RESOURCE_DIR);
+    llvm::SmallString<128> P = llvm::sys::path::parent_path(Executable);
+    llvm::sys::path::append(P, "..");
```

Comment:
Where are we now? Is this some approximation of the install prefix?

Reply:
`P` is the install prefix if iwyu itself. In `AbsoluteResourceDir` we're building the absolute path to the final resource directory. If it's absolute before, good; otherwise we're interpreting it relative to the install directory.